Preprocessing/Utils/AppendNF.ecl

```diff
@@ -0,0 +1,31 @@
+IMPORT $.^.^ as ML_Core;
+
+NumericField := ML_Core.Types.NumericField;
+
+/*##############################################################################
+## HPCC SYSTEMS software Copyright (C) 2020 HPCC Systems®.  All rights reserved.
+############################################################################## */
+
+/**
+ * Appends a numeric field dataset to another
```

Comment:
It is not clear whether this appends vertically, by adjusting the ids, or horizontally by adjusting number field.  Please clarify